sync/landing.py

```diff
@@ -617,20 +617,14 @@ def next_try_push(self, retry=False):
             return
 
         latest_try_push = self.latest_try_push
-        stability = False
+        stability = True
```

Comment:
I don't think we want to change the landing code here. We still need to do two try pushes there.

Reply:
Ahh I see, I was under the impression we also wanted that changed. Will revert!